src/mscorlib/src/System/Text/Encoding.cs

```diff
@@ -85,7 +85,11 @@ namespace System.Text
     [Serializable]
     public abstract class Encoding : ICloneable
     {
-        private static Encoding defaultEncoding;
+        // For netcore we use UTF8 as default encoding since ANSI isn't available
+        private static readonly UTF8Encoding.UTF8EncodingSealed s_defaultEncoding  = new UTF8Encoding.UTF8EncodingSealed(encoderShouldEmitUTF8Identifier: false);
+
+        // Returns an encoding for the system's current ANSI code page.
```

Comment:
Just moved comment

Comment above qualifies
`For netcore we use UTF8 as default encoding since ANSI isn't available`